packages/fxa-auth-server/lib/push.js

```diff
@@ -371,93 +350,131 @@ module.exports = function (log, db, config) {
      */
     async sendPush(uid, devices, reason, options = {}) {
       devices = filterSupportedDevices(options.data, devices);
-      const events = pushReasonsToEvents[reason];
-      if (!events) {
+      if (!PUSH_REASONS.has(reason)) {
         throw `Unknown push reason: ${reason}`;
       }
       // There's no spec-compliant way to error out as a result of having
       // too many devices to notify.  For now, just log metrics about it.
       if (devices.length > MAX_ACTIVE_DEVICES) {
-        reportPushError(new Error(ERR_TOO_MANY_DEVICES), uid, null);
+        log.warn(LOG_OP_TOO_MANY_DEVICES, { uid });
```

Comment:
Here and elsewhere, I've moved from logging these at `error` level and dropped it to `warn` instead. They don't represent anything that we've done wrong so ISTM they shouldn't be logged as errors. But, I'm very open to being convinced otherwise.